projects/vdk-plugins/vdk-ingest-http/src/vdk/plugin/ingest_http/ingest_http_plugin.py

```diff
@@ -8,16 +8,33 @@
 from vdk.api.plugin.hook_markers import hookimpl
 from vdk.internal.builtin_plugins.ingestion.ingester_base import IIngesterPlugin
 from vdk.internal.builtin_plugins.run.job_context import JobContext
+from vdk.internal.core.config import ConfigurationBuilder
 from vdk.plugin.ingest_http.ingest_over_http import IngestOverHttp
 
-
 log = logging.getLogger(__name__)
 
 
+@hookimpl
+def vdk_configure(config_builder: ConfigurationBuilder) -> None:
+    """
+    Here we define what configuration settings are needed for Greenplum with reasonable defaults
+    """
+    config_builder.add(
+        key="INGEST_OVER_HTTP_COMPRESSION_THRESHOLD_BYTES",
+        default_value=None,
+        description="Threshold used for determining if the payload will be compressed or not",
```

Comment:
Might be nice to mention in the description what would happen if the value of the threshold is left None.

Reply:
I've updated the description, lets see if self-explanatory enough?